### ISSUE TITLE
Narrator is not announcing tooltip information to users: A11y_.NET Core_Edit Examiner_Immediate Window_Screen reader.

### ISSUE THREAD

**Issue by keerthiduvvuri** [dotnet/wpf#9175]
### GIT Hub Tags
#AI Limited; #A11yTCS; #A11ySev2; #.NET Core; #E2E-.NET Core-WIN32-WPF-May2024; #Narrator; #WIN32; #WCAG1.3.1; #Info and Relationship; #FTP; #Win11; #Tooltip; #A11yMAS;

### Environment and OS details:
Windows 11 Enterprise version 23H2
OS Build:22631.3593

### Reproduction Steps

1. Launch Edit Examiner Demo Application from the extracted Zip File.
2. Edit Examiner Window gets opened.
3. Tab Navigate to "Immediate Window".
4. Turn on narrator and observe that Narrator is not announcing tooltip information to users.

### Expected behavior

Narrator should be announcing tooltip information to users.

### Actual behavior

Narrator is not announcing tooltip information to users.
Note: Issue is Not repro with NVDA

[Narrator not announcing tooltip info.webm](https://github.com/dotnet/wpf/assets/98728920/138f80fe-e349-40e0-a02e-a54b6bdb8e3c)

<img width="960" alt="Tool Tip Information is announced by NVDA" src="https://github.com/user-attachments/assets/7bc0682d-a6b1-4586-af42-701fcfb634e8">

### Impact

Screen reader users will face difficulty if Narrator is not announcing tooltip information to users.

**Comment by anjali-wpf** [comment]
@keerthiduvvuri - Duplicate issue. Please refer issue #[5036](https://github.com/dotnet/wpf/issues/5036)

**Comment by Yash14j** [comment]
GitHubTags:#Rev:yaja;

**Comment by merriemcgaw** [comment]
@Yash14j can you please close this since it is a duplicate?

**Comment by keerthiduvvuri** [comment]
Hi @merriemcgaw This bug is logged for Narrator, And this bug https://github.com/dotnet/wpf/issues/5036 is logged that NVDA is not announcing tooltip, but the issue is Fixed with NVDA.

Issue is Reproducing with Narrator, so Narrator bug should be fixed, we will not be closing this bug till the issue is fixed with Narrator. Thanks!

#Regressed:07-29-2024;

**Comment by keerthiduvvuri** [comment]
Discussed this bug with SME and closing as Duplicate.

#Regressed:07-29-2024; #Closed;

**Comment by merriemcgaw** [comment]
Thank you!
